src/Mod/Part/App/CMakeLists.txt

```diff
@@ -495,6 +509,7 @@ SET(Part_SRCS
     ${HLRBRepPy_SRCS}
     ${ShapeFixPy_SRCS}
     ${ShapeUpgradePy_SRCS}
+    ${FCBRepAlgoAPI_SRCS}
```

Comment:
In order to get builds with enabled PCH (precompiled headers) right `FCBRepAlgoAPI_SRCS` shouldn't be added to `Part_SRCS` but directly to `add_library`, i.e.:
`add_library(Part SHARED ${Part_SRCS} ${FCBRepAlgoAPI_SRCS})`

Reply:
done - however I don't have MSVC so I can't really fully test this - I made a cmake change so you can now force cmake -DFREECAD_USE_PCH=True when compiling on linux and that works - I hope that's enough.